scripts/prep_local_devel_env.sh

```diff
@@ -180,8 +180,16 @@ echo "Sleeping for a few seconds to avoid issues with broker not being able to t
 echo "Appears like there is a delay of when we create the asb-etcd route and when it is available for use"
 sleep 5
 
-etcd_route=`oc get route asb-etcd -n ${ASB_PROJECT} -o=jsonpath=\'\{.spec.host\}\'`
-echo "etcd route is at: ${etcd_route}"
+if [ -z "$LOCAL_ETCD" ]; then
```

Comment:
I think you want `if [ "$LOCAL_ETCD" == "true" ]; ` and flip the conditions only run the local one if it is set to true. else do the `oc get`